Fix README

I updated to go1.11.5 just to run this, and got...

`TENDERMINT_RPC="ws://bnsd.hugnet.iov.one" go run cmd/collector/main.go`

```
pkg/metrics/tendermint.go:12:2: cannot find package "github.com/gorilla/websocket" in any of:
        /usr/lib/go-1.11/src/github.com/gorilla/websocket (from $GOROOT)
        /home/ethan/go/src/github.com/gorilla/websocket (from $GOPATH)
pkg/metrics/pq.go:9:2: cannot find package "github.com/lib/pq" in any of:
        /usr/lib/go-1.11/src/github.com/lib/pq (from $GOROOT)
        /home/ethan/go/src/github.com/lib/pq (from $GOPATH)
```

And the cli args are just wrong

--- a/README.md
+++ b/README.md
@@ -1,5 +1,7 @@
 Requires go1.11+
 
+You must also `export GO111MODULE=on` in your environment to use the go modules feature.
+
 # Run locally
 
 For local development you can use a local Postgres instance and any Tendermint
@@ -11,6 +13,7 @@ $ docker run -it --rm -e POSTGRES_PASSWORD='' -p 5432:5432 postgres:alpine
 
 # Run collector. Default configuration is expected to work for local
 # development. If needed it can be changed via environment variables.
-$ TENDERMINT_RPC="TODO" \
+$ TENDERMINT_WS_URI="wss://bns.hugnet.iov.one/websocket" \
+  POSTGRES_URI="postgresql://postgres@localhost:5432" \
     go run cmd/collector/main.go
 ```

